src/main/kotlin/lotto/util/Constant.kt

```diff
@@ -0,0 +1,7 @@
+package lotto.util
+
+object Constant {
+    val LOTTO_NUM_RANGE=(1..45)
+    const val LOTTO_LEN=6
+    const val LOTTO_PRICE=1000.0
```

Comment:
```suggestion
    val LOTTO_NUM_RANGE = (1..45)
    const val LOTTO_LEN = 6
    const val LOTTO_PRICE = 1000.0
```
커밋 전에 IDE 에서 제공하는 reformat 기능을 한번 실행해주시는 습관을 들여주시면 좋습니다!

Reply:
Constant 는 너무 광범위한 역할을 할 수 있는 object 인 것 같아요.
실제로 사용되거나 어울리는 model 내부에 선언해주는 것은 어떨까요?

Reply:
모델에 선언을 하게 되면 서로 다른 클래스들끼리 의존성이 생긴다고 생각하였습니다.
ex> 로또에 선언한 경우 WinningLotto, Lottos, outputView등에서 활용 됨
따라서 어떤 모델의 내부보다는 네이밍을 로또에 관한 상수들로 수정하여 한정하게 되었습니다 괜찮을까요?